Dockerfile

```diff
@@ -40,8 +41,7 @@ COPY --from=actionlint /usr/local/bin/actionlint /usr/bin/
 RUN pip install pipenv \
     && pipenv install --system --deploy --ignore-pipfile \
     && apk upgrade \
-    && apk --no-cache add go \
```

Comment:
`go` was added in https://github.com/SeisoLLC/goat/pull/61 and doesn't seem to be needed until we readd go linters